charts/avalanche/templates/svc-avalanche.yaml

```diff
@@ -1,10 +1,9 @@
-# This file and its contents are licensed under the Apache License 2.0.
-# Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
-
+{{- range $n := until (int .Values.fakePrometheusTargetCount) }}
+{{- with $ -}}
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "avalanche.fullname" . }}
+  name: {{ include "avalanche.fullname" . }}-{{ $n }}
```

Comment:
We are using `serviceName: {{ include "avalanche.fullname" . }}` in the spec of the StatefulSet, this change will probably break the stable network IDs had previously. I think we can add another headless service with `name: {{ include "avalanche.fullname" . }}` to avoid breaking this behaviour. WDYT?

Reply:
We need to keep in mind that this headless service is excluded from the ServiceMonitor, so that we don't end up increasing the load data unintentionally. Maybe we can add an additional label to all scrapable services and add the same in `selector` section of ServiceMonitor.